<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Login {
    background: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1,
                                stop: 0 #1e3c72, stop: 0.5 #2a5298, stop: 1 #1e3c72);
}

QLabel {
    background: transparent;
}

QLabel#titleLabel {
    color: white;
    font-size: 28px;
    font-weight: bold;
    margin-top: 10px;
}

QLabel#subtitleLabel {
    color: rgba(255, 255, 255, 0.8);
    font-size: 14px;
    margin-bottom: 20px;
}

QPushButton#signInButton {
    background: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 0,
                                stop: 0 #1e3c72, stop: 1 #2a5298);
    color: white;
    border: 2px solid #FFD700;
    border-radius: 25px;
    padding: 12px;
    font-size: 16px;
    font-weight: bold;
}

QPushButton#signInButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 0,
                                stop: 0 #FFD700, stop: 1 #FFA500);
    color: #1e3c72;
    border: 2px solid #FFA500;
}

QPushButton#signInButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 0,
                                stop: 0 #FFA500, stop: 1 #FF8C00);
    color: white;
}

QPushButton#signUpButton {
    background: rgba(255, 255, 255, 0.1);
    color: white;
    border: 2px solid rgba(255, 255, 255, 0.3);
    border-radius: 25px;
    padding: 12px;
    font-size: 16px;
    font-weight: bold;
}

QPushButton#signUpButton:hover {
    background: rgba(255, 215, 0, 0.2);
    border: 2px solid #FFD700;
    color: #FFD700;
}

QPushButton#signUpButton:pressed {
    background: rgba(255, 215, 0, 0.3);
    border: 2px solid #FFA500;
}

QPushButton#forgotpButton {
    background: transparent;
    color: #FFD700;
    border: 2px solid transparent;
    border-radius: 25px;
    padding: 8px;
    font-size: 14px;
    font-weight: bold;
    text-decoration: underline;
}

QPushButton#forgotpButton:hover {
    background: rgba(255, 215, 0, 0.1);
    color: #FFA500;
    border: 2px solid rgba(255, 215, 0, 0.3);
    text-decoration: none;
}

QPushButton#forgotpButton:pressed {
    background: rgba(255, 215, 0, 0.2);
    color: #FF8C00;
    border: 2px solid rgba(255, 215, 0, 0.5);
}

QLineEdit {
    background: rgba(255, 255, 255, 0.15);
    border: 2px solid rgba(255, 255, 255, 0.3);
    border-radius: 25px;
    padding: 12px 20px;
    color: white;
    font-size: 14px;
    selection-background-color: #FFD700;
}

QLineEdit:focus {
    border: 2px solid #FFD700;
    background: rgba(255, 255, 255, 0.2);
}

QLineEdit::placeholder {
    color: rgba(255, 255, 255, 0.6);
}

QLineEdit:hover {
    border: 2px solid rgba(255, 215, 0, 0.5);
}

QLabel#logoLabel {
    background: transparent;
    border: none;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="logoLayout">
     <item>
      <widget class="QLabel" name="logoLabel">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>images/logo.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="styleSheet">
      <string notr="true">font: 700 italic 25pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Welcome Back</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="subtitleLabel">
     <property name="text">
      <string>Sign in to continue your journey</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="formLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <property name="leftMargin">
      <number>30</number>
     </property>
     <property name="rightMargin">
      <number>30</number>
     </property>
     <item>
      <widget class="QLabel" name="emailLabel">
       <property name="styleSheet">
        <string notr="true">color: #FFD700; font-size: 12px; font-weight: bold; padding-left: 20px;</string>
       </property>
       <property name="text">
        <string>EMAIL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="emailInput">
       <property name="placeholderText">
        <string>Enter your email</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="passwordLabel">
       <property name="styleSheet">
        <string notr="true">color: #FFD700; font-size: 12px; font-weight: bold; padding-left: 20px;</string>
       </property>
       <property name="text">
        <string>PASSWORD</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="passwordInput">
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Enter your password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="signInButton">
       <property name="text">
        <string>Sign In</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="signUpButton">
       <property name="text">
        <string>Create Account</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="orLabel">
       <property name="styleSheet">
        <string notr="true">color: rgba(255, 215, 0, 0.7); font-size: 12px;</string>
       </property>
       <property name="text">
        <string>or</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="forgotpButton">
       <property name="text">
        <string>Forgot Password?</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>